api/src/models/events.js

```diff
@@ -1,14 +1,26 @@
-const EventModel = (sequelize, Sequelize) => {
-  return sequelize.define("events", {
-    id: { type: Sequelize.UUID, primaryKey: true, validate: { isUUID: 4 } },
-    name: { type: Sequelize.STRING, allowNull: false },
-    date: { type: Sequelize.DATE, allowNull: false },
-    mainImageUrl: { type: Sequelize.STRING },
-    mainColor: { type: Sequelize.STRING },
-    logoUrl: { type: Sequelize.STRING },
-    secretKey: { type: Sequelize.STRING },
-    slug: { type: Sequelize.STRING }
-  });
-};
-
-export default EventModel;
+import mongoose from "mongoose";
+import uuid from "uuid";
+
+const Schema = mongoose.Schema;
+
+const eventSchema = new Schema({
+  eventId: {
```

Comment:
if you use `_id` I think it takes care of the indexing, uniqueness and what not. 
So I think you can just do `_id: { binData: Buffer, required: true }` and that way we don't have to use a custom name for it.